packages/java-spring/src/spring/components/spring-project.tsx

```diff
@@ -0,0 +1,30 @@
+import { ProjectDirectory, ProjectDirectoryProps, useProject } from "@alloy-js/java";
+
+export interface SpringProjectProps extends ProjectDirectoryProps {}
+
+export function SpringProject(props: SpringProjectProps) {
+  return (
+    <ProjectDirectory {...props}>
+      <SpringDependencies />
+      {props.children}
+    </ProjectDirectory>
+  );
+}
+
+/**
+ * Setup spring dependencies
+ */
+function SpringDependencies() {
+  const project = useProject();
+
+  project.scope.addDependency({
+    groupId: "org.springframework.boot",
+    artifactId: "spring-boot-starter-web",
+    version: "3.3.3",
```

Comment:
Can we make this version configurable? Users should be able to specify the spring version they want to use when they run `npx tsp compile --emit java-spring --option "@java-spring.version=3.3.5"`